modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFileEntryMetadataLocalServiceTest.java

```diff
@@ -243,6 +275,67 @@ public void testGetNoStructuresFileEntryMetadatas() throws Exception {
 		).build();
 	}
 
+	private DLFileEntry _getFileEntry(
+			long groupId, long fileEntryTypeId, ServiceContext serviceContext,
+			Map<String, DDMFormValues> ddmFormValuesMap)
+		throws Exception {
+
+		ByteArrayInputStream byteArrayInputStream = new ByteArrayInputStream(
+			TestDataConstants.TEST_BYTE_ARRAY);
+
+		return _dlFileEntryLocalService.addFileEntry(
+			null, TestPropsValues.getUserId(), groupId, groupId,
+			DLFolderConstants.DEFAULT_PARENT_FOLDER_ID,
+			RandomTestUtil.randomString(), null, RandomTestUtil.randomString(),
+			null, null, fileEntryTypeId, ddmFormValuesMap, null,
+			byteArrayInputStream, byteArrayInputStream.available(), null, null,
```

Comment:
Replace the "available" call by TestDataConstants.TEST_BYTE_ARRAY.length. The semantics of "available" are much more involved that a simple byte count (see https://docs.oracle.com/javase/7/docs/api/java/io/InputStream.html#available()).

Reply:
since we do not need the byteArrayInputStream I will inline the _addFileEntry/_getFileEntry